src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
That's a very good point. The controls-component currently holds the clock, ticks on requestAnimationFrame, and handles movement which is also kind of wrong I suppose.

In the game protocol, when a creature moves, what kind of stuff is sent? A timestamp and a target location? I'm just wondering whether there's anything that is not necessarily animation related but still needs to tick (somehow).